blocks/share-bar-block/features/share-bar/default.jsx

```diff
@@ -60,30 +48,26 @@ const share = {
 	},
 };
 
-function getLogoComponent(type) {
-	switch (type) {
-		case "facebook":
-			return <FacebookIcon fill="#4267B2" />;
-		case "linkedIn":
-			return <LinkedInAltIcon fill="#2867B2" />;
-		case "pinterest":
-			return <PinterestAltIcon fill="#BD081C" />;
-		case "twitter":
-			return <TwitterIcon fill="#1DA1F2" />;
-		default:
-		case "email":
-			return <EnvelopeIcon fill="#C72A22" />;
-	}
-}
+const getLogoComponent = {
+	facebook: <Icon name="Facebook" className={`${BLOCK_CLASS_NAME}__facebook`} />,
+	linkedIn: <Icon name="LinkedIn" className={`${BLOCK_CLASS_NAME}__linkedin`} />,
+	pinterest: <Icon name="Pinterest" className={`${BLOCK_CLASS_NAME}__pinterest`} />,
+	twitter: <Icon name="Twitter" className={`${BLOCK_CLASS_NAME}__twitter`} />,
+	email: <Icon name="Envelope" className={`${BLOCK_CLASS_NAME}__email`} />,
+};
 
 const ShareBarContainer = () => {
 	const {
 		customFields,
-		globalContent: { headlines: { basic: headlineString = "" }, website_url: websiteUrl = "" } = {},
+		globalContent: {
+			headlines: { basic: headlineString = "" },
+			website_url: websiteUrl = "",
+		},
```

Comment:
was the fallback `= {}` removed on purpose?

Reply:
@JackHowa : Yes, but it was correct to restore. Thank you for the suggested commit. I've included that change and restored the `={}` fallback.